Makefile.am

```diff
@@ -230,9 +230,9 @@ rpmvar_DATA =
 
 install-exec-hook:
 	@rm -f $(DESTDIR)$(bindir)/rpmquery
-	@LN_S@ ../../bin/rpm $(DESTDIR)$(bindir)/rpmquery
+	@LN_S@ rpm $(DESTDIR)$(bindir)/rpmquery
```

Comment:
This is a problem here. I have no idea where "rpm" exists in this symlink creation call, since it's not forcing it to be `/bin/rpm`.

When you're building and installing for OS X, is `rpm` still being installed to `/bin`?